backend/graphql/submission.graphql

```diff
@@ -10,6 +10,8 @@ type Submission {
     users: [User!]! @belongsToMany
     files: [SubmissionFile]! @hasMany
     pivot: SubmissionUser
+    status: Int!
```

Comment:
I strongly suggest adding a graphql enum type for this.

Reply:
This has been refactored as advised. This value is now represented as an enum.